internal/namespaces/k8s/v1beta4/custom_kubeconfig_uninstall.go

```diff
@@ -0,0 +1,126 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+	"gopkg.in/yaml.v2"
+)
+
+type k8sKubeconfigUninstallRequest struct {
+	ClusterID string
+}
+
+func k8sKubeconfigUninstallCommand() *core.Command {
+	return &core.Command{
+		Short:     `Uninstall a kubeconfig`,
+		Long:      `Remove specified cluster from kubeconfig file specified by the KUBECONFIG env, if empty it will default to $HOME/.kube/config.`,
+		Namespace: "k8s",
+		Verb:      "uninstall",
+		Resource:  "kubeconfig",
+		ArgsType:  reflect.TypeOf(k8sKubeconfigUninstallRequest{}),
+		ArgSpecs: core.ArgSpecs{
+			{
+				Name:       "cluster-id",
+				Short:      "Cluster ID from which to uninstall the kubeconfig",
+				Required:   true,
+				Positional: true,
+			},
+		},
+		Run: k8sKubeconfigUninstallRun,
+	}
+}
+
+func k8sKubeconfigUninstallRun(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+	request := argsI.(*k8sKubeconfigUninstallRequest)
+
+	// get the path to write the wanted kubeconfig on disk
```

Comment:
aren't we doing the same thing in `install` ? 

can we put this duplicated code in a common function ?